HeterogeneousCore/CUDAUtilities/interface/prefixScan.h

```diff
@@ -41,9 +41,9 @@ namespace cms {
   namespace cuda {
 
     // limited to 32*32 elements....
-    template <typename T>
-    __host__ __device__ __forceinline__ void blockPrefixScan(T const* __restrict__ ci,
-                                                             T* __restrict__ co,
+    template <typename VT, typename T>
+    __host__ __device__ __forceinline__ void blockPrefixScan(VT const* ci,
+                                                             VT* co,
```

Comment:
is `VT` supposed to be either `T` or `volatile T` ?

Reply:
yes,at least in this contest